addons/im_livechat/tests/chatbot_common.py

```diff
@@ -9,6 +9,7 @@ class ChatbotCase(MailCommon, common.HttpCase):
     @classmethod
     def setUpClass(cls):
         super(ChatbotCase, cls).setUpClass()
+        cls.maxDiff = None
```

Comment:
It looks like debug code

Reply:
Not limited to dev, it helps with better errors on runbot without having to run it locally. And has no impact when there is no test failure so I never understood why it's not the default.